test_ols.py

```diff
@@ -0,0 +1,69 @@
+from fastapi.testclient import TestClient
+
+from unittest.mock import MagicMock
+
+from ols import app
+
+client = TestClient(app)
```

Comment:
you were wondering what to classify this test as before..... since it is standing up the server and invoking it, but not using a real backend LLM, i'd say it's technically an "integration test", so we should probably call it that in the make target and CI job.

but again we can follow up on that after we land this.

Reply:
That sounds good to me. However, with that reasoning, every endpoint test on FastAPI will then be an "integration" test. I don't know if this is good or bad.